clearcut_detection_backend/services/landcover.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+from shutil import copyfile
+from utils import download_without_progress, fetch_file_from_zip, fetch_all_from_zip
+from requests.exceptions import (HTTPError, InvalidURL, ConnectionError)
+from zipfile import (BadZipFile, LargeZipFile)
+
+
+class Landcover:
+    tif = 'E020N60_ProbaV_LC100_epoch2015_global_v2.0.2_forest-type-layer_EPSG-4326.tif'
+
+    def __init__(self):
+        self.data_path = Path('./data')
+        self.landcover_path = self.data_path / 'landcover'
+        self.landcover_path.mkdir(parents=True, exist_ok=True)
+        # self.tif = 'E020N60_ProbaV_LC100_epoch2015_global_v2.0.2_forest-type-layer_EPSG-4326.tif'
+        print(self.landcover_path)
+        self.forest_tiff = self.landcover_path / 'forest.tiff'
+        print(self.forest_tiff)
+
+    @staticmethod
+    def download_landcover(url):
+        """
+        download file from url
+        :param url: str
+        :return: BytesIO
+        """
+        file = None
+        try:
+            file = download_without_progress(url)
+        except (HTTPError, InvalidURL, ConnectionError, ConnectionError) as e:
+            # TODO separate ConnectionError, this type of exception must be written to logs
+            print('e2 =', e)
```

Comment:
don't use print
use logging imported and configured in settings

Reply:
ok